### ISSUE TITLE
HTML Parsing issue on site

### ISSUE THREAD

**Issue by akester** [apache/incubator-pagespeed-mod#1458]
On http://bokehdevelopment.com, mod_pagespeed will only render a blank page.   Per [this](https://developers.google.com/speed/pagespeed/module/faq#broken) troubleshooting, disabling mod_pagespeed fixes the site, but re-enabling it with no filters causes the issue.

Pagespeed is disabled for the site currently, but the module is still on the server so you can test with various query parameters.

mod_pagespeed version is 1.12.34.1-0, Apache version is 2.4.6-45.el7.centos, running on CentOS 7.2.1511

**Comment by ciarans** [comment]
How are you caching the rendered data? Is you cache writable?

**Comment by akester** [comment]
We are using the default file cache at /var/cache/mod_pagespeed.  It appears to be writable and has data from that site and others stored there.

**Comment by Lofesa** [comment]
Hi @akester 
You make a redirect (301 permanent) from http://bokehdevelopment.com/properties to http://bokehdevelopment.com/properties/ how do you make that? have you tried the try_files directive?
Both pages are served whith Cache-control = "max-age=0"  and NPS needs resources tobe public cacheable.
Yuou need to allow your domain as far you use api.bokehdevelopment.com, this is done with " pagespeed Domain *.example.org;" 
At this point it seems you have no enabled NPS in your site, when you enable it ("pagespeed on;") you can see a header like that "x-page-speed:1.12.34.1-0", if you don´t see it, NPS is not working.

I tried http://bokehdevelopment.com/?PageSpeed=on and get the blank page, but don´t see the x-page-speed header.

Whats is showing in your server logs files? whats your configuration?

**Comment by akester** [comment]
We've disabled the module on that site since enabling it breaks the site.  In the response headers, there looks like some other headers are missing, specifically the X-Powered-By header set by Plesk, along with the pagespeed header when enabling the pagespeed module.

All the logs don't show any errors, just normal activity when making a request that causes the error, neither the nginx reverse proxy or the apache server.

We've also increased the max age, I would think that the module would still load the page at least, but I could be wrong.

**Comment by Lofesa** [comment]
If you use ngix as a proxy cache you must configure MPS with downstream cache, see it [here](https://developers.google.com/speed/pagespeed/module/downstream-caching)

Have you tried w/o the proxy?

P.D. I have reviewed your site and in the blank page I got when add /?PagSpeed=on I see that cache-control header changes and is like mod_pagespeed put in html files when are not fully optimized: cache-control:max-age=0, no-cache, s-maxage=10, seems like the mod_pagespeed start working but stop w/o finish their work. What you see at /pagespeed_global_admin/message_history?
